.gitignore

```diff
@@ -117,7 +117,6 @@ dmypy.json
 # editor
 .vscode
 .idea
```

Comment:
<img width="740" alt="CleanShot 2023-10-30 at 07 07 55@2x" src="https://github.com/koaning/scikit-lego/assets/1019791/016ed77e-734c-4ea4-8612-ad377aa805ab">

Isn't there a mkdoctrings setting to parse this?

Reply:
Yes, you can try with mkdocs.yaml, line 61 `docstring_style: sphinx`. However notice that it would not entirely solve the problem as the docstrings follow a "mixed" style. E.g. `linear_model.BaseScipyMinimizeRegressor` is documented in numpy style.

#586 addresses this by standardizing all the docstring to numpydoc (and rendering with mkdocs)